### DISCUSSION TITLE
Move homebrew to different directory in alternative installation

### DISCUSSION THREAD

**Discussion by blogabe** [Homebrew/discussions#597] [Everyday usage]
I have homebrew installed in a particular directory. I want to move it to a new location. Is there any feasible way to do this short of uninstalling all the formulae/casks/homebrew and reinstalling them after setting up homebrew in the new location?

**Comment by jonchang** [comment]
You can't really move Homebrew like this without uninstalling and reinstalling since there's certain issues around binary relocation that we don't support, but `brew bundle` might help you with the migration: 

https://github.com/homebrew/homebrew-bundle
